### ISSUE TITLE
Greenclip doen't work with Qt and Java applications

### ISSUE THREAD

**Issue by mssdvd** [erebe/greenclip#30]
I can't paste old snippets from the clipboard to Qt(e.g. qBittorent) and Java(e.g. Pycharm) applications but Greenclip works well with Gtk+ applications(e.g. chromium, emacs, termite...).

Config:
`Config {maxHistoryLength = 500, historyPath = "/home/davide/.cache/greenclip.history", staticHistoryPath = "/home/davide/.cache/greenclip.staticHistory", usePrimarySelectionAsInput = False}
`

I launch it with this: `rofi -modi "clipboard:greenclip print" -show clipboard`

Computer setup:
Archlinux, i3

**Comment by DmitriK** [comment]
I think I have this issue as well. Also Arch+i3. The issue occurs when I use greenclip to select a previous entry in the history. Trying to paste into qt applications after that doesn't work. What I can do is paste it into something else (I usually use Firefox address bar as it's close), re-copy it (which does not create a new history entry, so I don't think the data is different), and then pasting into qt applications works.

Greenclip exec line is `rofi -modi "clipboard:greenclip print" -show clipboard -run-command '{cmd}'`. I don't remember editing the greenclip config file, so it should be the defaults.

**Comment by theLogicJB** [comment]
I have the same issue like @DmitriK described.

**Comment by erebe** [comment]
Thanks for the detailed steps @DmitriK 
Indeed there was a bug in the 2.1 version (which was the official one)

The bug has been fixed in the 3.0-beta version, which I made the official one now as it is pretty stable.
If you are using Archlinux, just do a yaourt -Sy rofi-greenclip and you will get the new version.
It will delete your current config and cache, as there was some major rework between the 2.1 and 3.0.
Doing a greenclip --help should show the version 3.0.

Let me know if you still have the issue with the new version.
